models/safe/zkevm/safe_zkevm_schema.yml

```diff
@@ -0,0 +1,158 @@
+version: 2
+
+models:
+  - name: safe_zkevm_safes
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'zkevm']
+    description: "Safe addresses"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "The blockchain on which the Safe is deployed"
+      - &address
+        name: address
+        description: "Safe contract address"
+      - &creation_version
+        name: creation_version
+        description: "Version of initially created safe"
+      - &block_date
+        name: block_date
+      - &block_month
+        name: block_month
+      - &creation_time
+        name: creation_time
+        description: "Date/time of safe creation"
+      - &tx_hash
+        name: tx_hash
+
+  - name: safe_zkevm_matic_transfers
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'transfers', 'zkevm']
+    description: "MATIC transfers for safes"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - tx_hash
+            - trace_address
+            - amount_raw
```

Comment:
this test should match the unique keys [here](https://github.com/duneanalytics/spellbook/pull/5758/files#diff-3b4dccafa6414cb9bdf7730e46593d2571efbb83d1d6c7b44282dcb2d1ec6277R7)